src/client/VZSettings.tsx

```diff
@@ -77,6 +93,44 @@ export const VZSettings = ({
             Select a color theme for the editor
           </Form.Text>
         </Form.Group>
+
+        <Form.Group className="mb-3" controlId="formFork">
+          <Form.Label>Font</Form.Label>
+
+          <select
+            className="form-select"
+            onChange={handleFontChange}
+          >
+            {fonts.map((font) => (
+              <option key={font} value={font}>
```

Comment:
This does not show the previously selected option when you close the modal then re-open it.

Reply:
Indeed, it is correct! Now that the first version is ready, the next steps become clear.

I created a new issue to track the persistence: https://github.com/vizhub-core/vzcode/issues/562

Reply:
Oh but this comment was related to just the modal closing and re-opening, within a single session. I solved it in https://github.com/vizhub-core/vzcode/pull/559/files#diff-3cc8b6f15ff7e81275b96ef7bc6d64e13b89785525e651ccc6800b5da75aefa9R156